torch/csrc/api/include/torch/nn/modules/embedding.h

```diff
@@ -12,20 +12,32 @@ namespace nn {
 
 /// Options for the `Embedding` module.
 struct TORCH_API EmbeddingOptions {
-  EmbeddingOptions(int64_t count, int64_t dimension);
-  /// The number of embeddings (number of rows in the table).
-  TORCH_ARG(int64_t, count);
-  /// The size of each embedding vector (number of columns in the table).
-  TORCH_ARG(int64_t, dimension);
+  EmbeddingOptions(int64_t num_embeddings, int64_t embedding_dim);
+  // The number of embeddings (number of rows in the table).
+  TORCH_ARG(int64_t, num_embeddings);
+  // The size of each embedding vector (number of columns in the table).
+  TORCH_ARG(int64_t, embedding_dim);
+  // If given, pads the output with the embedding vector at :attr:`padding_idx (initialized to zeros) whenever it encounters the index.
+  TORCH_ARG(c10::optional<int64_t>, padding_idx)=c10::nullopt;
+  // If given, each embedding vector with norm larger than :attr:`max_norm` is renormalized to have norm :attr:`max_norm`.
+  TORCH_ARG(c10::optional<float>, max_norm)=c10::nullopt;
```

Comment:
We might not need to specify `c10::nullopt` for `padding_idx` and `max_norm`, because the default value of c10::optional should already be `c10::nullopt`.